jsonnet/dashboards/monitoring.libsonnet

```diff
@@ -46,14 +42,14 @@ local getGridY(offset, index) =
   */
   std.floor((index * panelWidth) / rowWidth) * panelHeight + offset;
 
-local getClusterRowGridY(numOfClusters) =
+local getClusterRowGridY(numOfClusters, panelWidth, panelHeight) =
```

Comment:
sizes of cluster and host statsPanels can differ because of templating and therefore for following template:
```
  layerL0:
    k8s:
      main:
        panel:
          gridPos:
            w: 10
            h: 10
```
dashboard will look like this (title for `Host Monitoring` row is below panels) :
![image](https://user-images.githubusercontent.com/72123207/107573250-8f44bd00-6bed-11eb-91c9-86ea6fc6843e.png)
But its pretty unlikely case, I dont know if its neccessary to address this issue.

Reply:
This problem is more complex than we thought. I found out that computing of coordinates (`getGridX`, `getGridY`) only works if `panel.gridPos.w % rowWidth == 0`. But we can merge this PR and address this problem in another issue.

Reply:
fixed